spec/factories/accessories.rb

```diff
@@ -7,6 +7,20 @@
     after(:create) do |instrument, evaluator|
       instrument.accessories << evaluator.accessory
     end
+
+    trait :not_reservation_only do
```

Comment:
Because `:setup_instrument` is the parent and you've added the `:not_reservation_only` trait there, I believe you can just call `not_reservation_only` here and it should work.